File: backoffice/FondaBackEnd/Exceptions/AddFavoriteRestaurantFondaWebApiControllerException.cs
using System;

namespace com.ds201625.fonda.FondaBackEnd.Exceptions
{
    /// <summary>
    /// Clase AddFavoriteRestaurantFondaWebApiControllerException
    /// </summary>
    public class AddFavoriteRestaurantFondaWebApiControllerException : FondaBackendException
    {
        public AddFavoriteRestaurantFondaWebApiControllerException  () : base() {	}

		public AddFavoriteRestaurantFondaWebApiControllerException  (string message) : base(message) {	}

        public AddFavoriteRestaurantFondaWebApiControllerException(string message, Exception InnerException)
			: base(message, InnerException) {	}
    }
}

File: frontoffice/app/src/main/java/com/ds201625/fonda/data_access/retrofit_client/exceptions/AddFavoriteRestaurantFondaWebApiControllerException.java
package com.ds201625.fonda.data_access.retrofit_client.exceptions;

import com.ds201625.fonda.domains.FondaBaseException;

/**
 * Clase AddFavoriteRestaurantFondaWebApiControllerException
 * clase de excepcion de tipo web api controller
 */
public class AddFavoriteRestaurantFondaWebApiControllerException extends FondaBaseException {

    public AddFavoriteRestaurantFondaWebApiControllerException() {
    }

    public AddFavoriteRestaurantFondaWebApiControllerException(String detailMessage) {
        super(detailMessage);
    }

    public AddFavoriteRestaurantFondaWebApiControllerException
                                             (String detailMessage, Throwable throwable) {
        super(detailMessage, throwable);
    }

    public AddFavoriteRestaurantFondaWebApiControllerException(Throwable throwable) {
        super(throwable);
    }

}
